ssz/README.md

```diff
@@ -1,21 +1,549 @@
-# simpleserialize (ssz)
+# simpleserialize (ssz) [WIP]
 
-This is a **work-in-progress** crate designed to perform the "simpleserialize"
-serialization described by Vitalik Buterin. The method is tentatively intended
-for use in the Ethereum Beacon Chain.
+This is currently a ***Work In Progress*** crate.
+
+
+SimpleSerialize is a serialization protocol described by Vitalik Buterin. The
+method is tentatively intended for use in the Ethereum Beacon Chain as
+described in the [Ethereum 2.1 Spec](https://notes.ethereum.org/s/Syj3QZSxm).
 
 There are two primary sources for this spec, and they are presently
```

Comment:
I think we should indicate that we are following the beacon chain spec and that we completely disregard the other spec. In other words, communicate that the beacon chain spec is canonical.